### PR TITLE
RIA-21054 Importing SOAP API collections

### PR DESCRIPTION
**Pull request by rysiek-precikowski**

### CODE REVIEW

File: src/main/java/com/smartbear/postman/PostmanImporter.java [hunk 1]
```diff
@@ -135,40 +126,34 @@ public WsdlProject importPostmanCollection(WorkspaceImpl workspace, String fileP
                     return null;
                 }
                 project.setDescription(postmanCollection.getDescription());
-                List<PostmanCollection.Request> requests = postmanCollection.getRequests();
-                for (PostmanCollection.Request request : requests) {
+                List<Request> requests = postmanCollection.getRequests();
+
+                SoapServiceCreator soapServiceCreator = new SoapServiceCreator(project);
```
Comment (marharyta-panska) [inline src/main/java/com/smartbear/postman/PostmanImporter.java:131]: Maybe we should move the creation of this service into `if (request.isSoap()) ` block?
I don't see it being used anywhere else inside this method

**Comment by marharyta-panska** [comment]
Are we planning to replace `AddRequestToTestCaseAction` with SilentAction implementation as for Rest requests? As we discussed, if user has e.g. 30 tests, test step window will open 30 times.

**Comment by rysiek-precikowski** [comment]
I think that we want to change it, but I'm not sure if it should be done in this pr or we should create a new task for it.

File: src/main/java/com/smartbear/postman/PostmanImporter.java (hunk 1 above)
Comment (rysiek-precikowski) [reply src/main/java/com/smartbear/postman/PostmanImporter.java:131]: Good catch, I'll move it

**Review by rszpakSB — APPROVED** [review]

**Pull request merged**